libs/server/Storage/Session/MainStore/MainStoreOps.cs

```diff
@@ -2,7 +2,10 @@
 // Licensed under the MIT license.
 
 using System;
+using System.Buffers;
+using System.Collections.Generic;
 using System.Diagnostics;
+using System.Linq;
```

Comment:
Unnecessary using

Reply:
It's used to perform the reverse operation in the list